### DISCUSSION TITLE
Possibility to choose different directory for "interp" folder

### DISCUSSION THREAD

**Discussion by SchwarzFuchs** [n00mkrad/flowframes#111] [Ideas]
I think this feature can seriously improve interpolation speed for people who use HDDs and have at least 2 of them. If the program uses two different HDDs for reading and writing at the same time istead of the one HDD, the speed of work can be significantly higher.

**Comment by n00mkrad** [comment]
Settings -> Application -> Temporary Folder Location

**Reply by SchwarzFuchs** [reply]
Moving the whole temporary folder won't give any perfomance impact because it moves "frames" folder too. My proposition is to have "interp" folder on drive №2 and all other folders from temporary folder on drive №1. This way read and write operation will be normally balanced between the two drives. Reading and writing at the same time seriousle severely degrades performance of any HDD

**Reply by HiImCris** [reply]
Is there a way to install it with insufficient space on disk C: and later change temp folder location?
